inst/info/not_transferred.json

```diff
@@ -0,0 +1,83 @@
+[
```

Comment:
Maybe there could be a better filename. The file contains metadata about repos that were not transferred or that store the R package in a subdir.